.eslintrc

```diff
@@ -138,7 +138,7 @@
     "no-underscore-dangle": "off",
     "one-var": ["error", "never"],
     "padded-blocks": ["off", "never"],
-    "semi": ["error", "always"],
+    "semi": ["error", "never"],
```

Comment:
Changes to the `.eslintrc` file should be addressed in [our styleguide.](https://github.com/redradix/javascript-style-guide/blob/master/linters/.eslintrc)

It is true that our linter configuration doesn't match our style guide as of now, but it's being reworked as we speak. We will bring the stable version to the base app as soon as its ready.